src/screens/WebView/PrivacyPolicies.js

```diff
@@ -29,7 +30,7 @@ export default withInteractionsManaged(React.memo(PrivacyPolicies));
 const styles = StyleSheet.create({
   container: {
     flex: 1,
-    backgroundColor: '#fff',
+    backgroundColor: COLORS.white,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from a hardcoded color value to using a constant from a centralized theme file is a good practice. It improves maintainability and consistency across the application.
```diff
-    backgroundColor: '#fff',
+    backgroundColor: COLORS.white,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->